Sources/DiscoverKit/View/RecommendationsView.swift

```diff
@@ -29,8 +34,8 @@ struct RecommendationsView: View {
                             view.frame(width: RecommendationsView.maxReadableWidth, alignment: .center)
                         }
             }
+            .navigationTitle(Text("Today's Top Picks"))
         }
-        .searchable(text: $viewModel.term)
```

Comment:
was it decided to remove the search from here?

Reply:
Based on discussions with search PRD and designs, it seems not needed at the time of the removal. And I just confirmed with Moiz that it will live in another section so to be removed from Discover tab.